### PR TITLE
Add permissive CORS headers to assets

### PR DESCRIPTION
**Pull request by timdorr**

### CODE REVIEW

**Comment by timdorr** [comment]
tag @sb8244

File: unicorn/templates/default/nginx_unicorn_web_app.erb [hunk 1]
```diff
@@ -30,6 +30,14 @@ server {
     expires 1y;
     add_header Cache-Control public;
 
+    # If you don't find the filename in the static files
+    # Then request it from the unicorn server
+    try_files $uri @unicorn;
```
Comment (sb8244) [inline unicorn/templates/default/nginx_unicorn_web_app.erb]: I think this would require rails to have serve_static_assets = true

Comment (timdorr) [inline unicorn/templates/default/nginx_unicorn_web_app.erb]: This just means a 404 falls back to the app server if it's in `/assets/`

**Comment by sb8244** [comment]
:+1:

**Comment by timdorr** [comment]
Wait a tick. This isn't going to work...

**Comment by sb8244** [comment]
:+1: but no idea if this will work

**Pull request merged**